ocaml/xapi/xapi_pool.ml

```diff
@@ -2120,6 +2120,7 @@ let is_slave ~__context ~host:_ =
   debug
     "About to kick the database connection to make sure it's still working..." ;
   let (_ : bool) =
+    Atomic.set Master_connection.in_call_from_coordinator true ;
```

Comment:
Shouldn't it be set to false after the database check is done?

Reply:
The indicator is a boolean value. Setting to false may conflict with other callers (although there are no such callers at the moment).